plugins/coral-plugin-offtopic/client/components/OffTopicFilter.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from './styles.css';
+
+export default class OffTopicFilter extends React.Component {
+
+  tag = 'OFF_TOPIC';
+  className = 'offTopicComment';
```

Comment:
for global classnames, we've been doing something like `coral-plugin-off-topic-comment`. Trying to keep it simple.